app/assets/stylesheets/card.scss

```diff
@@ -0,0 +1,32 @@
+.card {
+  background: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.